src/modules/pages/LoginPage/LoginForm.scss

```diff
@@ -0,0 +1,45 @@
+@import 'index.scss';
+
+$form-width: 530px;
+$form-height: 387px;
+$logo-width: 100px;
+$logo-height: 92px;
+$form-padding-top: 70px;
+
+.login-form {
+    position: absolute;
+    width: $form-width;
+    height: $form-height;
+    padding: $form-padding-top $hf-padding-big $hf-padding-big;
+    background-color: $hf-color-white;
+    border: $hf-border;
+    border-radius: $hf-border-radius;
+    box-sizing: border-box;
+    @include alignCenter($form-width, $form-height);
+
+    &__logo {
+        position: absolute;
+        width: $logo-width;
+        height: $logo-height;
+        top: 0;
+        margin-top: -$logo-height/2;
+        @include alignCenterX($logo-width);
+    }
+
+    &__greeting {
```

Comment:
@Vxplus there is one class for this, but that class is the part of `Row` component and is applying as a modifier.

In order to implement that way I need to wrap the paragraph with `Row` component(smthg like this):

```javascript
<Row alignCenter><p>Enter your HelloFresh credentials to login to <strong>Janus Gateway</strong></p></Row>
```

I think this current way is much simpler